docs/file-based-configuration.md

```diff
@@ -0,0 +1,64 @@
+# File-based Configuration
+
+> **Status:** [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md)
+> For more information about the OpenTelemetry configuration specification, see:
+> **[File-based configuration documentation](https://opentelemetry.io/docs/specs/otel/configuration/sdk/)**  
+
+You can configure OpenTelemetry using a YAML file. This method is disabled
+by default and must be explicitly enabled.
+
+To enable file-based configuration, set the following environment variable:
+
+```bash
+OTEL_EXPERIMENTAL_FILE_BASED_CONFIGURATION_ENABLED=true
+```
+
+By default, the value is false.
+
+You can also specify the configuration file path (default: config.yaml):
+
+```bash
+OTEL_EXPERIMENTAL_CONFIG_FILE=/path/to/config.yaml
+```
+
+In your config file you can use environment variables in the format `${ENVIRONMENT_VARIABLE}`
```

Comment:
What about `${env:ENVIRONMENT_VARIABLE}` syntax?

I also suggest referring to https://github.com/open-telemetry/opentelemetry-specification/blob/v1.49.0/specification/configuration/data-model.md#yaml-file-format for more information